.github/workflows/build_executables.yml

```diff
@@ -23,9 +23,7 @@ jobs:
 
       - name: 📦 Install Dependencies
         shell: bash
-        run: |
-          flutter pub global activate very_good_cli
-          very_good packages get --recursive
```

Comment:
Why no longer use it?

Reply:
It's totally unnecessary and just bloats the CI, especially in this workflow a simple `flutter pub get` does exactly what we need.